<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filters</class>
 <widget class="QDialog" name="Filters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>373</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Apply Filters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="verticalFrame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>381</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>241</height>
           </size>
          </property>
          <property name="title">
           <string>Clipper Filter</string>
          </property>
          <widget class="QCheckBox" name="Clipper_Filter">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>ClipperFilter On/Off</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>181</width>
             <height>191</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>191</height>
            </size>
           </property>
           <property name="title">
            <string>Origin</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>20</y>
              <width>151</width>
              <height>161</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="ClipperFliter">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>X-Adjustment</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="X_Origin">
                 <property name="minimum">
                  <number>-9999999</number>
                 </property>
                 <property name="maximum">
                  <number>9999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Y-Adjustment</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="Y_Origin">
                 <property name="minimum">
                  <number>-9999999</number>
                 </property>
                 <property name="maximum">
                  <number>9999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Z-Adjustmet</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="Z_Origin">
                 <property name="minimum">
                  <number>-9999999</number>
                 </property>
                 <property name="maximum">
                  <number>9999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>50</y>
             <width>181</width>
             <height>191</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>191</height>
            </size>
           </property>
           <property name="title">
            <string>Normal</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>19</y>
              <width>154</width>
              <height>161</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>X-Adjustment</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="X_Normal">
                 <property name="minimum">
                  <number>-99999999</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                 <property name="value">
                  <number>-1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Y-Adjustment</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="Y_Normal">
                 <property name="minimum">
                  <number>-99999999</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Z-Adjustmet</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="Z_Normal">
                 <property name="minimum">
                  <number>-99999999</number>
                 </property>
                 <property name="maximum">
                  <number>99999999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>81</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>181</width>
            <height>81</height>
           </size>
          </property>
          <property name="title">
           <string>Shrink Filter</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="ShrinkFilter">
            <item alignment="Qt::AlignHCenter">
             <widget class="QCheckBox" name="Shrink_Filter">
              <property name="text">
               <string>Shrink Filter On/Off</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Scale</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Scale">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>90</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Close">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
